.github/dependabot.yml

```diff
@@ -120,6 +120,9 @@ updates:
       # Don't try to auto-update any DSpace dependencies
       - dependency-name: "org.dspace:*"
       - dependency-name: "org.dspace.*:*"
+	  # Ignore major/minor updates for Hibernate. Only patch updates can be automated.
+	  - dependency-name: "org.hibernate.*:*"
+		update-types: ["version-update:semver-major", "version-update:semver-minor"]
```

Comment:
@alanorth : I think the alignment on all of these is incorrect.  They need to be lined up with the dependencies listed above, as this is YAML.  YAML is super picky about alignment :) 

Also, I think "update-types" should be aligned directly under "dependency-name", similar to the one just below this (where we ignore *all* major updates for all dependencies)

Reply:
Ugh, mixing tabs and spaces. To be fair it was aligned in vim. :smile: Fixed now.